### PR TITLE
build: release v6.44.1

### PR DESCRIPTION
**Pull request by KenLSM**
## Solution

**Breaking Changes** 

- [x] No - this PR is backwards compatible  

**Features**:
- Updated copy for Payment Email Field from `Email` to `Email Address`
- Updated copy for Payment Invoice `Date Paid` to `Invoice Date`

## Before & After Screenshots

**BEFORE**:

**AFTER**:

Screenshot 2023-04-25 at 6 11 11 PM
Screenshot 2023-04-25 at 6 11 24 PM

### CODE REVIEW

File: src/app/modules/payments/stripe.utils.ts [hunk 1]
```diff
@@ -360,6 +360,7 @@ export const convertToInvoiceFormat = (
       /Receipt (#[0-9-]+)/,
       `Invoice $1<br /><br />GST Reg No: ${GST_REG_NO}<br />Address: ${ADDRESS}`,
     )
+    .replace(/Date paid/g, 'Invoice Date')
```
Comment (KenLSM) [inline src/app/modules/payments/stripe.utils.ts:363]: Using `/g` here as there are two instance of `Date paid`

File: frontend/src/features/public-form/components/FormPaymentPreview/FormPaymentPreview.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ export const FormPaymentPreview = ({
   const sectionColor = useSectionColor(colorTheme)
   const emailFieldSchema: VerifiableEmailFieldSchema = {
     ...(getFieldCreationMeta(BasicField.Email) as EmailFieldBase),
+    title: 'Email Address',
     _id: PAYMENT_CONTACT_FIELD_ID,
     description: 'Proof of payment will be sent to this email',
```
Comment (LinHuiqing) [inline frontend/src/features/public-form/components/FormPaymentPreview/FormPaymentPreview.tsx]: JTC wanted to change this to "For delivery of invoice", right? What is our stance on this? (Not sure if I missed some discussion in office? 😅)

Comment (KenLSM) [reply frontend/src/features/public-form/components/FormPaymentPreview/FormPaymentPreview.tsx]: Missed it! Updated in 66823d32

**Review by foochifa — APPROVED** [review]
tested and lgtm thanks ken

**Pull request merged**